proto/c1/connector/v2/connector.proto

```diff
@@ -105,3 +107,46 @@ message ConnectorServiceValidateRequest {}
 message ConnectorServiceValidateResponse {
   repeated google.protobuf.Any annotations = 1;
 }
+
+message ConnectorAccountCreationSchema {
+  map<string, Field> field_map = 1;
+
+  message Field {
+    string display_name = 1;
+    bool required = 2;
+    string description = 3;
+    string placeholder = 4;
+    int32 order = 5;
+
+    bool deprecated = 6;
+    oneof field {
+      StringField str = 100;
+      BoolField bool = 101;
+      StringListField str_list = 102;
+      IntField i = 103;
+      MapField m = 104;
+    }
+  }
+
+  message StringField {
+    bool secret = 1;
```

Comment:
Not sure this actually makes sense since these are configured as expressions - it's probably wrong in the source. I'll leave it to your discretion whether you'd like to remove it.

Reply:
What does this attribute mean? The value isn't unencrypted, right? Is it just "please don't display this"?

Reply:
I lean towards deleting it. If you want to keep it around, I think it'd be best to rename it to "sensitive" or "hidden" or something, along with a comment explaining that it's just a display hint.